### PR TITLE
chore: support groups in `lightdash-dbt-2.0.json`

### PR DESCRIPTION
**Pull request by yu-iskw**
Closes: xxx

### Description:

The changes is to support groups in `lightdash-dbt-2.0.json`. 

https://docs.lightdash.com/references/dimensions/#groups

I have checked the JSON schema with [check-jsonschema](https://github.com/python-jsonschema/check-jsonschema) locally.

```
$ check-jsonschema --check-metaschema packages/common/src/schemas/json/lightdash-dbt-2.0.json -v
ok -- validation done
The following files were checked:
  packages/common/src/schemas/json/lightdash-dbt-2.0.json
```

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [x] I have reviewed the code
- [x] I understand that "request changes" will block this PR from merging

### CODE REVIEW

File: packages/common/src/schemas/json/lightdash-dbt-2.0.json [hunk 1]
```diff
@@ -76,6 +76,24 @@
                                 "type": "string",
                                 "enum": ["index", "label"]
                             },
+                            "group_details": {
+                                "type": "object",
+                                "patternProperties": {
+                                    "^[a-zA-Z0-9_]+$": {
```
Comment (yu-iskw) [inline packages/common/src/schemas/json/lightdash-dbt-2.0.json:82]: Should it be `^[a-z0-9_]+$` as the `metrics`?

File: packages/common/src/schemas/json/lightdash-dbt-2.0.json [hunk 2]
```diff
@@ -325,6 +343,12 @@
                                                             ]
                                                         }
                                                     ]
+                                                },
+                                                "groups": {
+                                                    "type": "array",
+                                                    "items": {
+                                                        "type": "string"
+                                                    }
```
Comment (yu-iskw) [inline packages/common/src/schemas/json/lightdash-dbt-2.0.json:351]: We can also set `maxItems` to 2 for the current specification.

```json
                                                "groups": {
                                                    "type": "array",
                                                    "items": {
                                                        "type": "string"
                                                    },
                                                    "maxItems": 2
                                                }
```

File: packages/common/src/schemas/json/lightdash-dbt-2.0.json [hunk 3]
```diff
@@ -325,6 +343,13 @@
                                                             ]
                                                         }
                                                     ]
+                                                },
+                                                "groups": {
+                                                    "type": "array",
+                                                    "items": {
+                                                        "type": "string",
+                                                        "minLength": 1
+                                                    }
```
Comment (yu-iskw) [inline packages/common/src/schemas/json/lightdash-dbt-2.0.json:352]: We can also set `maxItems` to 2 for the current specification.

```json
                                                "groups": {
                                                    "type": "array",
                                                    "items": {
                                                        "type": "string",
                                                        "minLength": 1
                                                    },
                                                    "maxItems": 2
                                                }
```

File: packages/common/src/schemas/json/lightdash-dbt-2.0.json (hunk 1 above)
Comment (joaoviana) [reply packages/common/src/schemas/json/lightdash-dbt-2.0.json:82]: let's go with this, yes!

File: packages/common/src/schemas/json/lightdash-dbt-2.0.json [hunk 4]
```diff
@@ -76,6 +76,24 @@
                                 "type": "string",
                                 "enum": ["index", "label"]
                             },
+                            "group_details": {
+                                "type": "object",
```
Comment (joaoviana) [inline packages/common/src/schemas/json/lightdash-dbt-2.0.json:80]: can you add this line please?
```
"description": "Set up group_details so you can group your dimensions and metrics in the sidebar using the groups parameter. You can create nested groups up to 2 levels",
```

File: packages/common/src/schemas/json/lightdash-dbt-2.0.json (hunk 3 above)
Comment (joaoviana) [reply packages/common/src/schemas/json/lightdash-dbt-2.0.json:366]: could you add that pls?
And to each `groups`, this as well: 

```
  "description": "Groups are used to group dimensions and metrics in the sidebar. You can create nested groups up to 2 levels",
```

**Review by joaoviana — CHANGES_REQUESTED** [review]
Looks great @yu-iskw , thx! Just added a few suggestions

**Review by joaoviana — APPROVED** [review]
pushed the changes, ty for contributing @yu-iskw 🙌

**Comment by yu-iskw** [comment]
@joaoviana thank you for the patch!

**Pull request merged**